lib/credentials.js

```diff
@@ -60,9 +91,38 @@ Credentials.fromExtendedPrivateKey = function(xPrivKey) {
   return x;
 };
 
-Credentials.fromExternalWalletPublicKey = function(xPubKey, source, index) {
+
+Credentials.keyFromMnemonic = function(words, passphrase, network) {
```

Comment:
This looks like an internal method, I think prepending a `_` would make the API cleaner.